spec/spec_helper.rb

```diff
@@ -13,7 +13,10 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+require 'factory_bot_rails'
```

Comment:
Do we need this requirement?

Reply:
without this requirement `uninitialized` constant FactoryBot`